libs/api-bot/default.nix

```diff
@@ -2,27 +2,89 @@
 # This file is generated by running hack/bin/generate-local-nix-packages.sh and
 # must be regenerated whenever local packages are added or removed, or
 # dependencies are added or removed.
-{ mkDerivation, aeson, ansi-terminal, api-client, async, attoparsec
-, base, base64-bytestring, bilge, bytestring, bytestring-conversion
-, cereal, containers, cryptobox-haskell, cryptonite, exceptions
-, filepath, gitignoreSource, HaskellNet, HaskellNet-SSL
-, http-client, imports, iso639, lib, memory, metrics-core, mime
-, monad-control, mwc-random, optparse-applicative, resource-pool
-, stm, text, time, tinylog, transformers-base, types-common
-, unordered-containers, uuid, vector
+{ mkDerivation
```

Comment:
could we not exclude these auto-generated files from the formatting? It's a bit useless to format these, no? +4,365 −1,060 seems unnecessary. We should format the files people write themselves.

Reply:
Alright, okay. 😃